### PR TITLE
Feature/save files response

### PR DESCRIPTION
**Pull request by stefanprokopdev**
Returns `fileId` from google storage and `originalFile` with all provided metadata from `saveFiles` method.

### CODE REVIEW

**Review by vlasy — APPROVED** [review]

File: src/lib/fileUploader/gcpBucket.ts [hunk 1]
```diff
@@ -68,7 +67,7 @@ export const saveFiles = (bucket: Bucket, files: File[], options: Options) => {
                         .pipe(remoteFile)
                         .on('error', reject)
                         .on('finish', resolve)
-                ).then(() => fileId);
+                ).then(() => ({ fileId, originalFile: file.originalFile }));
```
Comment (smolijar) [inline src/lib/fileUploader/gcpBucket.ts]: I think that using wait here would increase readability [#6](https://github.com/AckeeCZ/unicore/pull/6#discussion_r334331643)

Comment (smolijar) [reply src/lib/fileUploader/gcpBucket.ts]: Please just make sure that all the types work correctly. If I am not mistaken, the function now returns a new type (file). Test it works even when the types are not installed in the project using unicore (e.g. they are not just in dev dependencies).  [#6](https://github.com/AckeeCZ/unicore/pull/6#discussion_r334331643)

**Review by smolijar — COMMENTED** [review]
How is this related to [#6](https://github.com/AckeeCZ/unicore/pull/6#discussion_r334331643)?
If its the same changes, close one of them. :pray:

File: src/lib/fileUploader/gcpBucket.ts (hunk 1 above)
Comment (stefanprokopdev) [reply src/lib/fileUploader/gcpBucket.ts]: #6 is draft and will be open after we merge this PR

`.then` replaced with `await` and types tested

**Review by smolijar — APPROVED** [review]

**Pull request merged**